pathutil/pathutil.go

```diff
@@ -179,3 +179,10 @@ func NormalizedOSTempDirPath(tmpDirNamePrefix string) (retPth string, err error)
 	}
 	return
 }
+
+// GetFileName returns the name of the file from a given path or the name of the directory if it is a directory
+func GetFileName(path string) string {
+	file := filepath.Base(path)
```

Comment:
Note:
It's perfectly fine, but you can get the same result with:
```
{
    return strings.TrimRight(filepath.Base(path), filepath.Ext(path))
}
```

Reply:
TrimRight removes the last char of the filename as well, changed to TrimSuffix.